app/scenes/login_options/login_options.js

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PureComponent, PropTypes} from 'react';
+import {
+    Image,
+    Text,
+    View
+} from 'react-native';
+
+import Button from 'react-native-button';
+import FormattedText from 'app/components/formatted_text';
+import {GlobalStyles} from 'app/styles';
+
+import logo from 'assets/images/logo.png';
+
+export default class LoginOptions extends PureComponent {
+    static propTypes = {
+        actions: React.PropTypes.shape({
+            goToLogin: PropTypes.func.isRequired,
+            goToSaml: PropTypes.func.isRequired
+        }).isRequired,
+        config: PropTypes.object.isRequired,
+        license: PropTypes.object.isRequired
+    };
+
+    renderEmailOption = () => {
+        const config = this.props.config;
+        if (config.EnableSignInWithEmail === 'true' || config.EnableSignInWithUsername === 'true') {
+            return (
+                <Button
+                    key='email'
+                    onPress={this.props.actions.goToLogin}
+                    containerStyle={[GlobalStyles.signupButton, {backgroundColor: '#2389d7'}]}
+                >
+                    <FormattedText
+                        id='signup.email'
+                        defaultMessage='Email and Password'
```

Comment:
If SignInWithEmail is disabled but SignInWithUsername is enabled, we'd still be displaying "Email and Password" here. But oh well.